polylogue/_vendor/pyperclip.py

```
@@ -0,0 +1,22 @@
+
+from __future__ import annotations
+
+_CLIPBOARD = ""
+
+
+class PyperclipException(RuntimeError):
+    pass
+
+
+def copy(text: str) -> None:
+    global _CLIPBOARD
```

Ruff (0.15.9)

[warning] Using the global statement to update `_CLIPBOARD` is discouraged

(PLW0603)